src/cljs/main/broadfcui/page/workspace/summary/catalog/questions.cljs

```diff
@@ -131,6 +131,27 @@
                              :onClick #(apply swap! state update :attributes dissoc
                                               (library-utils/get-related-id+label-props library-schema property))})]]))])
 
+(defn- render-populate-typeahead [{:keys [prop property]}]
+  [:div {:style {:marginBottom "0.75em"}}
+   [comps/AutocompleteFilter
+    {:width "100%"
+     :ref "text-filter"
+     :placeholder (:inputHint prop)
+     :bloodhoundInfo {:url (str (config/api-url-root) "/api/library/populate/suggest/" (name property) "/")
+                      :cache false
+                      :prepare (fn [query settings]
+                                 (clj->js
+                                   (assoc (js->clj settings)
+                                     :headers {:Authorization (str "Bearer " (utils/get-access-token))}
+                                     :type "GET"
+                                     :url (str (aget settings "url") query)
+                                     )))}
+     :typeaheadDisplay (fn [result]
+                         (.text (js/$ (str "<div>" result "</div>"))))
```

Comment:
It's really suspicious to me that we're taking a thing, wrapping it in an element, then getting the text out.  I'm not sure what's going on under the covers, but I understand why you needed to do this, and we should investigate simplifying this.